bin/csv-filter

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env ruby
+
+require 'optparse'
+require 'csv'
+
+# Returns an array of converter names.
+# \String +s+ contains a comma-separated list of converter names;
+# each optionally begins with a colon.
+def parse_converters(s)
+  converters = []
+  s.split(',').each do |name|
+    name.sub(/^:/, '')
+    sym = name.to_sym
+    converters.push(sym)
+  end
+  converters
+end
+
+options = {}
+
+OptionParser.new do |p|
+
+  p.program_name = File.basename $0
+  p.version = CSV::VERSION
+  p.release = nil
+  p.summary_indent = ' ' * 4
+  p.banner = <<-EOF
+Usage: #{p.program_name} [options]
+
+  Reads and parses the CSV text content of $stdin' per the given input options.
+  From that content, generates CSV text per the given output options and writes
+  that text to $stdout.
+
+  Three options are similar for input and output:
+
+  - Row separator:
+  
+    - The input row separator is determined:
+  
+      - By option --input_row_sep, if given.
+      - Otherwise, by option --row_sep, if given.
+      - Otherwise, by auto-discovery.
+  
+    - The output row separator is determined:
+  
+      - By option --output_row_sep if given.
+      - Otherwise, by option --row_sep if given.
+      - Otherwise, as the newline character ("\\n").
+  
+  - Column separator:
+  
+    - The input column separator is determined:
+  
+      - By option --input_col_sep, if given.
+      - Otherwise, by option --col_sep``, if given.
+      - Otherwise, as the comma character (',').
+  
+    - The output column separator is determined:
+  
+      - By option --output_col_sep if given.
+      - Otherwise, by option --col_sep if given.
+      - Otherwise, as the comma character (',').
+  
+  - Quote character:
+  
+    - The input quote character is determined:
+  
+      - By option --input_quote_char, if given.
+      - Otherwise, by option --quote_char, if given.
+      - Otherwise, as the double-quote character ('"').
+  
+    - The output quote character is determined:
+  
+      - By option --output_quote_char, if given.
+      - Otherwise, by option --quote_char, if given.
+      - Otherwise, as the double-quote character ('"').
+EOF
+
+  p.separator(nil)
+  p.separator(
+    'Input Options: see https://docs.ruby-lang.org/en/3.3/CSV.html#class-CSV-label-Options+for+Parsing.'
+  )
+
+  p.separator(nil)
+  p.on('--converters STR',
```

Comment:
* Could you use `--name=LABEL` style not `--name LABEL`?
* Could you use better name than `STR`?

```suggestion
  p.on('--converters=CONVERTERS',
```

Reply:
Changed to equal-style for those options not commented out;  changed STR to something else.